modules/k3d/k3d.sh

```diff
@@ -82,18 +86,28 @@ delete() {
   fi
 }
 
+create_namespaces() {
+  namespaces=$(ack -h --nobreak "namespace: '.*'," lib/*/*.libsonnet | sed -n "s/namespace: '\(.*\)',/\1/ p" | awk '{$1=$1};1' | sort | uniq)
```

Comment:
```
tk env list --json | jq -r '.[].spec.namespace' | sort | uniq
```

Probablhy a more common way to achieve this these days (plus you won't run into issues with awk/sed differences between arch).

Reply:
nice, liking tanka the more i learn about it :)